Fixes bug "pdf problems with gv" (Bug #13200).
http://savannah.gnu.org/bugs/index.php?func=detailitem&item_id=13200

by  Markus Steinborn <msteinbo@uni-paderborn.de>,
    Student of Computer Science
    University of Paderborn.

Index: src/callbacks.c
===================================================================
--- src/callbacks.c	(Revision 25)
+++ src/callbacks.c	(Revision 26)
@@ -357,7 +357,11 @@
 
     BEGINMESSAGE(cb_checkFile)
     changed = check_file(((int)client_data));
-    if (changed==1) show_page(current_page,NULL);
+    if (changed==1)
+    {
+       cb_stopInterpreter(page,NULL,NULL);
+       show_page(current_page,NULL);
+    }
     ENDMESSAGE(cb_checkFile)
 }
 
